Source/WebCore/editing/ReplaceSelectionCommand.cpp

```diff
@@ -230,7 +230,7 @@ void ReplacementFragment::removeContentsWithSideEffects()
     auto end = descendantsOfType<Element>(*m_fragment).end();
     while (it != end) {
         Ref element = *it;
-        if (isScriptElement(element) || (is<HTMLStyleElement>(element) && element->getAttribute(classAttr) != WebKitMSOListQuirksStyle)
+        if (isScriptElement(element) || (is<HTMLStyleElement>(element) && element->getAttribute(classAttr) != AtomString(WebKitMSOListQuirksStyle))
```

Comment:
This is not a correct fix; it’s inefficient because it requires looking up the string in the AtomString table every time. The correct fix is to add this missing overload to `AtomString.h`:
`inline bool operator!=(const AtomString& a, ASCIILiteral b) { return !(a == b); }`

The reason @cdumez missed this is that in a modern compiler in C++20 mode, the `!=` function gets automatically generated. The version of clang we are using on Apple platforms is new enough to have this feature.

Reply:
Thank you for the review. I wasn't sure about the proposed change. Overloading `operator!=(const AtomString& a, ASCIILiteral b)` indeed fixed the build error.